code/modules/station_goals/dna_vault.dm

```diff
@@ -136,6 +136,10 @@ GLOBAL_LIST_INIT(non_simple_animals, typecacheof(list(/mob/living/carbon/human/m
 	invisibility = 101
 	var/obj/machinery/parent
 
+/obj/structure/filler/Destroy()
+	parent = null
+	..()
```

Comment:
Destroy returns a value so this needs to be `return ..()`.
The value returned will determine if the object gets queued for deletion, let alive or hard deleted etc.

Reply:
Still requires the `return ..()`